### PR TITLE
changed font to Montserrat

### PR DESCRIPTION
**Pull request by CamilingJS**
## Summary (1-2 sentences)
Switched out the default font to Monserrat. 
Changes were added in the tailwind.config.tsx file. 
Though, I'm not confident this is the best way to do it with Tailwind, but it somehow worked :)

## Screenshots/Recordings (if applicable)
image

### CODE REVIEW

**Comment by CamilingJS** [comment]
I believe I'm running into linting errors I'm unsure to fix lol

File: tailwind.config.ts [hunk 1]
```diff
@@ -10,6 +11,9 @@ export default {
         warm: '#E11D48',
         grayBackground: 'rgba(217, 217, 217, 0.25);',
       },
+      fontFamily: {
+        'sans': ['"Montserrat"', ...defaultTheme.fontFamily.sans],
```
Comment (andrelandgraf) [inline tailwind.config.ts:15]: Nice!

File: tailwind.config.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import type { Config } from 'tailwindcss';
+const defaultTheme = require('tailwindcss/defaultTheme');
```
Comment (andrelandgraf) [inline tailwind.config.ts:2]: I think you want to use `import` syntax here. Otherwise, ESLint complains!

File: tailwind.config.ts (hunk 1 above)
Comment (shami-dev) [reply tailwind.config.ts:15]: Should we use fallback fonts here?

**Comment by shami-dev** [comment]
> I believe I'm running into linting errors I'm unsure how to fix lol

Probably, we should use the 'sans' word without quotes:
fontFamily: {
        sans: ['"Montserrat"', ...defaultTheme.fontFamily.sans],

File: app/root.tsx [hunk 3]
```diff
@@ -30,12 +30,13 @@ export const meta: MetaFunction = ({ error }: any) => {
...
-    { rel: 'stylesheet', href: stylesheet },
-  ];
-};
+export const links: LinksFunction = () => [
+  { rel: 'icon', type: 'image/svg-xml', href: faviconAssetUrl },
+  { rel: 'stylesheet', href: stylesheet },
+  { rel: 'preconnect', href: 'https://fonts.googleapis.com'},
+  { rel: 'preconnect', href: 'https://fonts.gstatic.com', crossOrigin: 'anonymous' },
+  { rel: 'stylesheet', href: 'https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap' }, 
+];
```
Comment (shami-dev) [inline app/root.tsx:39]: Not a blocker: Should we add all of these font styles? If we need only three of them is it better to download them?🙂

**Review by andrelandgraf — APPROVED** [review]
LGTM :)

**Review by eiffelwong1 — APPROVED** [review]
LGTM, :shipit:

**Pull request merged**